Xamarin.Forms.Platform.WinRT/ListViewRenderer.cs

```diff
@@ -83,6 +83,12 @@ protected override void OnElementChanged(ElementChangedEventArgs<ListView> e)
 				// WinRT throws an exception if you set ItemsSource directly to a CVS, so bind it.
 				List.DataContext = new CollectionViewSource { Source = Element.ItemsSource, IsSourceGrouped = Element.IsGroupingEnabled };
 
+#if !WINDOWS_UWP
```

Comment:
I'm curious why it was already working in UWP but not WinRT. Were you able to determine why?